src/hooks/UrqlUseMutation.re

```diff
@@ -54,10 +54,63 @@ let useMutation = (~request) => {
     );
 
   let executeMutation =
-    React.useCallback1(
+    React.useCallback2(
       () => executeMutationJs(Some(request##variables)),
-      [|request##variables|],
+      (executeMutationJs, request##variables),
```

Comment:
It seemed like this should be in the dependency array, since it's a function returned from a hook and its identity is not guaranteed.